### PR TITLE
fix: [M3-7692] - VPC Action Buttons Incorrect Color in Dark Mode

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Fixes button colors of VPC action buttons (`Edit` and `Delete`)
- Relates to https://github.com/linode/manager/pull/10077

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-01-23 at 8 54 50 PM | Screenshot 2024-01-23 at 8 54 40 PM |

## How to test 🧪

- Go to VPC details page
- Check the colors of the `Edit` and `Delete` buttons in both light ☀️ and dark mode 🌑

## As an Author I have considered 🤔

- [x] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [x] 🔐 Removing all sensitive information from the code and PR description
- [x] 🚩 Using a feature flag to protect the release
- [x] 👣 Providing comprehensive reproduction steps
- [x] 📑 Providing or updating our documentation
- [x] 🕛 Scheduling a pair reviewing session
- [x] 📱 Providing mobile support
- [x] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/VPCs/VPCDetail/VPCDetail.styles.ts [hunk 1]
```diff
@@ -9,8 +9,8 @@ export const StyledActionButton = styled(Button, {
   label: 'StyledActionButton',
 })(({ theme }) => ({
   '&:hover': {
-    backgroundColor: theme.color.blueDTwhite,
-    color: theme.color.white,
+    backgroundColor: theme.color.blue,
+    color: '#fff',
```
Comment (dwiley-akamai) [inline packages/manager/src/features/VPCs/VPCDetail/VPCDetail.styles.ts:13]: Why doesn't `theme.color.white` work here? Looks like it's supposed to, L62 in `light.ts` is set to `#fff`

**Review by dwiley-akamai — APPROVED** [review]
Colors display as expected in dark mode ✅

The "Impact" field on the ticket 😭

File: packages/manager/src/features/VPCs/VPCDetail/VPCDetail.styles.ts (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/VPCs/VPCDetail/VPCDetail.styles.ts:13]: Apparently `theme.color.white` is kind of a lie. In dark mode it is not white

Screenshot 2024-01-24 at 10 32 30 AM

Comment (dwiley-akamai) [reply packages/manager/src/features/VPCs/VPCDetail/VPCDetail.styles.ts:13]: Ah, I see in `dark.ts` L57: `white: '#32363c'`

**Review by abailly-akamai — APPROVED** [review]
Thanks! fix works as expected 👍

**Pull request merged**